Assets/Scripts/Player/PlayerControls.cs

```diff
@@ -9,6 +9,15 @@ public class PlayerControls : MonoBehaviour
 	[SerializeField] float _interactCheckRadius = 0.2f;
 
 	PlayerActor _actor;
+
+	[Tooltip( "Probably straight up" )]
+	[SerializeField] Vector3 _respawnOffset = new Vector3();
+	[ReadOnly( "Respawn Position" )]
+	[SerializeField] Vector3 _respawnPosition = new Vector3();
+	Quaternion _respawnRotation = new Quaternion();
```

Comment:
I don't know if this is costly to have new Quaternion/Vector3 but  Quaternion.Identity or Vector3.Zero are probably more readable.